src/components/navMenu/navMenu.tsx

```diff
@@ -10,7 +10,7 @@ const NavMenu = ({ render }) => {
       aria-label={text}
       key={text}
       sx={{
-        mb: "0px",
+        marginBottom: "0px",
```

Comment:
Is this to be more verbose or was it an actual issue?

Reply:
to be more verbose/consistent across styling